new-cep.md

```diff
@@ -0,0 +1,94 @@
+
+[pilots_conda](https://user-images.githubusercontent.com/6708083/233211589-10bca344-141e-407a-8edc-ca39014605eb.png)
+
+### Pilot Responsibilities
+
+* Responsible and accountable for the release of a given conda (or another conda project) version.
+* Ensure that users are able to successfully update to the latest version and that no critical bugs or issues arise as part of the release.
+* Be the known go-to liaison if there is a potentially critical issue, and take point in addressing/fixing it.
+* Responsible for getting the co-pilot prepared to be a pilot by being a teacher and ensuring that they will be able to take over the process next time.
+
+### Co-pilot Responsibilities
+
+* Responsible for certain roles or activities for a given conda (or another project) version.
+* Be an extra set of eyes to help the pilot get to the correct outcome.
+* Help with communication with the team and the broader community.
+* Learn as an active participant in the process, not meant to simply be "on deck" for the next time.
+
+### Jump seat Responsibilities
+* None. This is simply for someone that wants to just be along for the ride and learn by observing. This is not meant to be a "backup" or "on deck" role. This is an intentional choice in order to prevent the "too many cooks in the kitchen" problem as even though there might be good intentions, it can be distracting and confusing to have too many people involved in the process. With all that said there is benefits to allowing someone to just sit in and observe, so this is a compromise as this isn't meant to be a secret process, but it is meant to be a focused process as issues can arise at any time and the pilot and co-pilot need to be able to focus on the task at hand.
+
+### Special Note for Previous Pilots
+
+* If you are coming from being the pilot on the previous release, then make sure to help and guide the current pilot if there are any questions that come up.
+
+## Motivation
+
+The proposed release process is necessary to ensure a high-quality, consistent, and efficient release cycle for conda and related projects. The pilot and co-pilot system encourages knowledge transfer, collaboration, and continuous improvement.
+
+## Rationale
+
+The pilot and co-pilot model has been proven to work well in other projects, ensuring that there is always at least one person with the necessary experience and knowledge to handle releases, while also providing opportunities for other team members to learn and grow. This inspiration came from seeing how PyCon US has run their conference where the same city runsthe conference twice in a row, with the second year providing an opportunity for the next years city to learn from the previous years city.
+
+## Alternatives
+
+1. Have a dedicated release manager role, but this would limit the opportunities for team members to learn and grow, and it may create a single point of failure if the release manager becomes unavailable.
```

Comment:
```suggestion
1. Have a dedicated release manager role, but this would limit the opportunities for the community at large to learn and grow, and it may create a single point of failure if the release manager becomes unavailable.
```